examples/ansible/roles/alp_reboot/tasks/start.yml

```diff
@@ -0,0 +1,41 @@
+- name: Enable lingering for the user
+  ansible.builtin.command:
+    cmd: "loginctl enable-linger {{ ansible_user }}"
```

Comment:
Note that this could be considered a potential security risk if left enabled, so you potentially should be disabling it again, if you had to enabled it, after successfully rebooting.

Also it might be a good idea to provide a comment block here explaining why lingering is needed?

Reply:
added a comment block explaining why lingering is needed and a cleanup task to disable lingering.